### ISSUE TITLE
Suspected vulnerabilities in dependencies

### ISSUE THREAD

**Issue by bssth** [ProtonMail/gopenpgp#279]
dependabot complains that some of your library dependencies have known vulnerabilities. This is about github.com/cloudflare/circl and golang.org/x/crypto

Proposes from bot:
- Update github.com/cloudflare/circl from 1.3.3 to 1.3.7
- Update golang.org/x/crypto from 0.7.0 to 0.17.0

..exactly the same as from Goland IDE. Is it possible to upgrade to versions that are considered secure?

**Comment by lubux** [comment]
Hi 👋 You could switch to the [2.8.0 pre-release](https://github.com/ProtonMail/gopenpgp/releases/tag/v2.8.0-alpha.0), which bumps the versions of the dependencies.

**Comment by bssth** [comment]
> Hi 👋 You could switch to the [2.8.0 pre-release](https://github.com/ProtonMail/gopenpgp/releases/tag/v2.8.0-alpha.0), which bumps the versions of the dependencies.

Hi! Kindly tell me if it is stable enough to use. Thanks for fast response!

**Comment by bssth** [comment]
Problem with circl gone, but I have another one:
![image](https://github.com/ProtonMail/gopenpgp/assets/6750722/6e537bcd-dde1-46b2-ac86-e6d18a75be2d)

**Comment by lubux** [comment]
> Hi! Kindly tell me if it is stable enough to use. 

Yes, the pre-release can be used. It adds support for the OpenPGP [crypto-refresh](https://datatracker.ietf.org/doc/draft-ietf-openpgp-crypto-refresh/) if enabled, which is not fully published yet. This is why it is still a pre-release. 

> Problem with circl gone, but I have another one:

GopenPGP does not rely on the SSH features in x/crypto, so it is fine:
https://github.com/golang/crypto/compare/v0.17.0...v0.23.0"

**Comment by bssth** [comment]
> GopenPGP does not rely on the SSH features in x/crypto, so it is fine: [golang/crypto@v0.17.0...v0.23.0](https://github.com/golang/crypto/compare/v0.17.0...v0.23.0)"

So it's not used, just indirect dependency of another dependency which is not used in your project?
